erpnext/accounts/doctype/account/account.py

```
@@ -340,17 +465,16 @@ def on_trash(self):
 		if self.check_gle_exists():
 			throw(_("Account with existing transaction can not be deleted"))
 
-		super(Account, self).on_trash(True)
+		super().on_trash(True)
 
 
 @frappe.whitelist()
 @frappe.validate_and_sanitize_search_inputs
 def get_parent_account(doctype, txt, searchfield, start, page_len, filters):
```

Ruff (0.14.5)

Unused function argument: `doctype`

(ARG001)